README.md

```diff
@@ -1 +1,120 @@
-# CodersCamp2021.Project.Fullstack
\ No newline at end of file
+
+# CodersCamp 2021 - Projekt Node.js
+
+<div align="center">
+
+[![Demo](https://img.shields.io/badge/-demo-green?logo=github)](https://coderscamp2021-hk-fullstack.herokuapp.com/)
+[![Figma](https://img.shields.io/badge/-brainstorming+wireframing-blueviolet?logo=figma)](https://www.figma.com/file/51gzaCTbilz6X9r0PZxmhX/Project.Fullstack?node-id=0%3A1)
+</div>
+
+<br/>
+
+<p>
+🍲🍝🍜 <strong>BierzCoChcesz</strong> 🍲🍝🍜 projekt skierowany do ludzi, którzy nie lubią gdy narzuca im się wybory jedzeniowe a jednocześnie zwracają uwagę na wartości odżywcze swoich posiłków.
+Aplikacja pozwala na skomponowanie przez użytkownika diety na cały dzień składającej się z dań z ulubionych restauracji. W praktyce to system partnerski zrzeszający najlepsze restauracje.</p>
```

Comment:
Opis wydaje mi się być dość ogólny, dla nas pewne rzeczy są oczywiste, ale dopisałabym jeszcze że jest to coś na zasadzie cateringu i dopowiedziała o co chodzi z tym zwracaniem uwagi na wartości odżywcze - że docelowo na bieżąco sumowane będą kalorie i makro.